### ISSUE TITLE
AMD version is missing!

### ISSUE THREAD

**Issue by minzak** [RavenCommunity/kawpowminer#47]
In release only NVIDIA version.
Where is AMD?

On README.md - I see 46 line:
**For AMD-only rigs please use the version with -amd tagged , cuda version wouldn't work for you rig.**

But where is AMD version?

P.S. build still not worked! https://github.com/RavenCommunity/kawpowminer/issues/42

**Comment by jlar0che** [comment]
Yeah, I can't get Kawpowminer working with AMD cards. It is true, there is no -amd tagged version available for download. Anyone out there available to help us with this?

**Comment by vinhphamh** [comment]
Same problem.  Can't get the windows binary version 1.2.3 to run on AMD.  I thought the binary was good for both AMD and NVIDIA because it doesn't say "cuda".  Can someone please shed some light on this?  Thanks.

**Comment by netlogica** [comment]
Same problem. Can't get Linux Ubuntu version for my AMD cards rig

**Comment by jeroz1** [comment]
Are you using the mandatory `kawpowminer -G` flag for AMD?
